Add explicit definitions from Geek Feminsim

ammeep opened the pull request:
Continuing from https://github.com/todogroup/opencodeofconduct/pull/8

I think any code of conduct should include specific definitions of harassment, and also outline in detail the steps for dealing with incidents.

Let me know what you think :sparkles:

fat16 committed:
We can have an exhaustive list

fat16 committed:
added consequences section

fat16 committed:
remove duplicate statement

fat16 committed:
loosen anti abuse team

While I do wish we could leave this in, I do understand that this might
be a requirement that prevents people from adopting the code of conduct
:cry:

caniszczyk commented:
Thanks for the PR @ammeep! 

Initially we were trying to aim for brevity while covering all of the bases. We tried the cover some of the explicit definitions in the "Be careful in the words that you choose:" section. 

The consequences section I think is redundant based on the first paragraph and even last sentence there: "Anyone who violates this code of conduct may be banned from the community."

Any other ideas :)? Thoughts @bkeepers?

vgturtle127 commented:
It says this is causing conflicts and merge conflicts must be resolved? That's weird, because, you know, it's just text? HALP... :stuck_out_tongue_winking_eye:

ammeep commented:
>Initially we were trying to aim for brevity while covering all of the bases.

Thanks for that context @caniszczyk :sparkles: 

How would you feel about removing attribution to Geek Feminism? One of the great things about the Geek Feminism code of conduct is that is is super explicit. By being brief, I don't think we are in the spirit of the original.

caniszczyk commented:
@ammeep we could remove the reference but this CoC was definitely inspired by all the ones listed in the thanks/attribution section. We looked at what were recommended CoCs and why [1]

How about this, I ooked to the Rust CoC for inspiration as a way to keep things shorter but also relevant to your request and in spirit with what GF considers a solid CoC [2]

How about a bullet that amends the "Be careful in the words that you choose" section with:

> Be careful in the words that you choose: we are a community of professionals, and we conduct ourselves professionally. Be kind to others. Do not insult or put down other participants. Harassment and other exclusionary behavior aren’t acceptable. We interpret the term "harassment" as including the definition in the [Geek Feminism Code of Conduct](http://geekfeminism.org/about/code-of-conduct/); if you have any lack of clarity about what might be included in that concept, please read their definition.

[1] - http://geekfeminism.wikia.com/wiki/Code_of_conduct_evaluations
[2] - http://www.rust-lang.org/conduct.html

caniszczyk commented:
@ammeep I came up with a PR here: https://github.com/todogroup/opencodeofconduct/pull/24

Let me know if that helps resolve the issue.

stephbwills commented:
@caniszczyk @ammeep Hello! I'm a writer at GitHub. I agree that brevity is a noble goal, but after reading through the PRs on this and hearing some reactions, I think it is worth explicitly listing what constitutes harassment or figuring out a way to ensure that the concerns of marginalized communities are represented in the CoC. As is, the CoC feels exclusionary without the list in Girl Geek Feminism. 

Perhaps there is a way we can fix this without sacrificing brevity, but I'm not sure hyperlinking does the trick. It does force dependency on whatever is behind the link. I think it would benefit us to own the content ourselves and stand behind it.

vgturtle127 commented:
Yeah, what @stephbwills said. I hate linking to stuff anyway. Same reason I never put links in my CSS for fonts, or link to external images. If sites go down, your site is still up. Yay!

Addvilz commented:
This PR is superfluos and has issues similar to to #24. CoC should protect everyone equally, without emphasis on one distinct group, marginalized or not.

stephbwills commented:
@Addvilz this PR proposes including a list defining types of harassment without linking to it, whereas #24 suggests linking to the list as a solution to the issues in this PR. 

The list in Geek Feminism is explicit about forms of harassment, which is helpful. It also defines areas of harassment against certain communities (the transgender community, for example) that people who aren't in those communities might just not understand or be aware of. Including the list (or a list) obviates any guesswork as to what does or does not constitute bad behavior. 

From a practical perspective, adapting the list for the CoC and having it live there also makes the content easier to maintain. Hope this helps explain the goal of this PR.

modality commented:
+1 to this PR.

Addvilz commented on index.md:
```diff
@@ -22,7 +22,35 @@ Our open source community strives to:
+- Unwelcome sexual attention
+- Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others
+- Continued one-on-one communication after requests to cease
+- Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect other GF members or other vulnerable people from intentional abuse
```
Reference to Geek Feminism should be removed or replaced with [COMMUNITY]

stephbwills replied:
Yah, could be: 

>  Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect people from intentional abuse

Addvilz commented:
@stephbwills I believe, even if it might be helpful, it is still superfluos and it faces the same issues as commented in #26. There exists a widely accepted coverall definition ref. in #24 and I believe a short and clear statement as the one linked can cover pretty much every type of harassment conceivable towards everyone equally and without emphasis on any particular group. In addition to that, it is stated in clear and common language, and does not requires in depth knowledge in specific terminology regarded to any specific group in particular, eliminating any indirect discrimination to anyone who might not be scholar enough in various terms used in this pull request.

Argument that someone outside a given community might or might not understand or be aware of something is opinionated and discussion about it can be endless. Even more, as stated before, a single sentence delivered in a clear and unambiguous way will deliver more value than endless paragraphs of specifics which require previous knowledge and is subject to debate. 

Otherwise this pull request should include definitions for all marginalized groups in existence, not just some select ones. 

Although the idea might be good and helpful, the execution leaves lot to be desired. One example would be that the construction of given types and other suggested content is constructed in almost rude, derogatory and overly restrictive language which might lead one to believe this to be code of enforcement for a particular viewpoint, not a general code of conduct.

fat16 committed:
add [COMMUNITY] place holder

fat16 committed:
:fire: remove loose terminology 

and reference explicit definitions

fat16 committed:
Merge branch 'gh-pages' into add-back-explicit-definitions

Conflicts:
	index.md

ammeep commented:
I've addressed the feedback with relation to explicit definitions of what constitutes and does not constitute harassment :sparkles: @bkeepers @caniszczyk @stephbwills how does this look?

stephbwills commented on index.md:
```diff
@@ -11,18 +11,38 @@ Our open source community strives to:
+Harassment includes:
+
+- Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, race, age, or religion
+- Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment.
```
Probs don't need the period at the end of this.

stephbwills commented on index.md:
```diff
@@ -11,18 +11,38 @@ Our open source community strives to:
+- Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment.
+- Deliberate misgendering or use of ‘dead’ or rejected names
+- Gratuitous or off-topic sexual images or behaviour  in spaces where they’re not appropriate
+- Physical contact and simulated physical contact (eg, textual descriptions like “*hug*” or “*backrub*”) without consent or after a request to stop.
```
Same comment as above, different period :grin:

stephbwills commented on index.md:
```diff
@@ -11,18 +11,38 @@ Our open source community strives to:
+- Stalking or following
+- Harassing photography or recording, including logging online activity for harassment purposes
+- Sustained disruption of discussion
+- Unwelcome sexual attention
```
If we're concerned about brevity, it seems like we can roll this one up with one of the bullet points above. Like: 

> Unwelcome sexual attention, including gratuitous or off-topic sexual images or behaviour in spaces where they’re not appropriate

Addvilz replied:
I second this, would make it broader and more clear. As a sidenote, I fail to imagine where such actions would be appropriate if taken in to context of professional environment.

Addvilz commented on index.md:
```diff
@@ -11,18 +11,38 @@ Our open source community strives to:
+
+- Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, race, age, or religion
+- Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment.
+- Deliberate misgendering or use of ‘dead’ or rejected names
```
Rephrase or expand in a clearer language. What does this "dead or rejected names" even mean?

Addvilz commented on index.md:
```diff
@@ -11,18 +11,38 @@ Our open source community strives to:
+
+Our open source community prioritizes marginalized people’s safety over privileged people’s comfort. The **[COMMUNITY]** community leaders will not act on complaints regarding:
+
+- ‘Reverse’ -isms, including ‘reverse racism,’ ‘reverse sexism,’ and ‘cisphobia’ (because these things don’t exist)
```
Highly opinionated and constitutes exclusionism of certain, valid issues (forced point of view).

bkeepers replied:
The feedback we've heard this week overwhelmingly has suggested adding reverse-isms. I agree that this is important to add, but "because these things don't exist" feels snarky and unnecessary.

Addvilz commented on index.md:
```diff
@@ -11,18 +11,38 @@ Our open source community strives to:
+- Deliberate misgendering or use of ‘dead’ or rejected names
+- Gratuitous or off-topic sexual images or behaviour  in spaces where they’re not appropriate
+- Physical contact and simulated physical contact (eg, textual descriptions like “*hug*” or “*backrub*”) without consent or after a request to stop.
+- Threats of violence
```
This could be expanded. For example, "Threats of violence, both physical and psychological"

Addvilz commented on index.md:
```diff
@@ -11,18 +11,38 @@ Our open source community strives to:
+
+Harassment includes:
+
+- Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, race, age, or religion
```
This list should include political affiliation and regional discrimination.

Addvilz commented on index.md:
```diff
@@ -11,18 +11,38 @@ Our open source community strives to:
+- Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect other [COMMUNITY] members or other vulnerable people from intentional abuse
+- Publication of non-harassing private communication
+
+Our open source community prioritizes marginalized people’s safety over privileged people’s comfort. The **[COMMUNITY]** community leaders will not act on complaints regarding:
```
The first statement is group-exclusive and can be interpreted as discrimination against particular groups. It should be either clarified, or removed.

bkeepers replied:
Same here as previously, can we remove the `[COMMUNITY]` placeholder here?

> We will not act on complaints regarding:

or

> Complaints regarding the following will not be acted upon:

Addvilz commented on index.md:
```diff
@@ -11,18 +11,38 @@ Our open source community strives to:
+
+- ‘Reverse’ -isms, including ‘reverse racism,’ ‘reverse sexism,’ and ‘cisphobia’ (because these things don’t exist)
+- Reasonable communication of boundaries, such as “leave me alone,” “go away,” or “I’m not discussing this with you.”
+- Refusal to explain or debate social justice concepts
```
Concepts are to be explained and debated, that is why they are concepts, not facts. I don't believe this has anything to do with CoC.

bkeepers replied:
@Addvilz This is only saying that the refusal to debate is not a violation of the code of conduct. I feel like that is reasonable to include.

Addvilz commented on index.md:
```diff
@@ -11,18 +11,38 @@ Our open source community strives to:
+* **Be careful in the words that you choose**: we are a community of professionals, and we conduct ourselves professionally. Be kind to others. Do not insult or put down other participants. Harassment (as defined bellow) and other exclusionary behavior aren't acceptable.
 * **When we disagree, try to understand why**: Disagreements, both social and technical, happen all the time. It is important that we resolve disagreements and differing views constructively. Remember that we’re different. The strength of our community comes from its diversity, people from a wide range of backgrounds. Different people have different perspectives on issues. Being unable to underst ...
 
-This code is not exhaustive or complete. It serves to distill our common understanding of a collaborative, shared environment, and goals. We expect it to be followed in spirit as much as in the letter.
```
I don't think this is to be deleted completely. The last sentence provides valuable guidance for the interpretation of the CoC.

Addvilz commented on index.md:
```diff
@@ -11,18 +11,38 @@ Our open source community strives to:
-This code is not exhaustive or complete. It serves to distill our common understanding of a collaborative, shared environment, and goals. We expect it to be followed in spirit as much as in the letter.
+## Definitions
+
+Harassment includes:
```
"but is not limited to"

jglovier commented:
>I think any code of conduct should include specific definitions of harassment, and also outline in detail the steps for dealing with incidents.

>The list in Geek Feminism is explicit about forms of harassment, which is helpful. It also defines areas of harassment against certain communities (the transgender community, for example) that people who aren't in those communities might just not understand or be aware of. Including the list (or a list) obviates any guesswork as to what does or does not constitute bad behavior.

@ammeep, @stephbwills, and others who've contributed here, huge THANK YOU. :heart: :metal: :sparkles: :bow: I think having a clear standard to measure against, and to be informed by is very important for the sake of community improvement. Having ambiguous language certainly affords freedom to penalize and take action against offenders in a very broad way, but doesn't specifically help people stop taking the offensive actions. By listing out examples, it provides a standard for people to judge their behavior by, and learn from.

bkeepers commented on index.md:
```diff
@@ -11,18 +11,38 @@ Our open source community strives to:
-  * Unwelcome sexual attention.
-  * Advocating for, or encouraging, any of the above behavior.
-  * Repeated harassment of others. In general, if someone asks you to stop, then stop.
+* **Be careful in the words that you choose**: we are a community of professionals, and we conduct ourselves professionally. Be kind to others. Do not insult or put down other participants. Harassment (as defined bellow) and other exclusionary behavior aren't acceptable.
```
typo on `bellow` here.

But I'm curious if that phrase is even needed (I have a bias against parenthetical statements in non-conversational prose).

bkeepers commented on index.md:
```diff
@@ -11,18 +11,38 @@ Our open source community strives to:
+- Unwelcome sexual attention
+- Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others
+- Continued one-on-one communication after requests to cease
+- Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect other [COMMUNITY] members or other vulnerable people from intentional abuse
```
Could we reword this so `[COMMUNITY]` is not needed here? Multiple occurrences just mean people have to remember to change all of them when copying the template, and if we can do without it, I think it makes it easier.  Maybe something like:

> Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect others from intentional abuse

bkeepers commented:
I'm :+1: on this direction. Brevity initially was a goal, but I don't think it's worth it at the cost of ambiguity.

caniszczyk commented on index.md:
```diff
@@ -42,7 +62,8 @@ include them as well. Your account of what occurred, and if you believe the inci
 After filing a report, a representative will contact you personally. If the person who is harassing you is part of the response team, they will recuse themselves from handling your incident. A representative will then review the incident, follow up with any additional questions, and make a decision as to how to respond. We will respect confidentiality requests for the purpose of protecting victim ...
 
-Anyone asked to stop unacceptable behavior is expected to comply immediately. If an individual engages in unacceptable behavior, the representative may take any action they deem appropriate, up to and including a permanent ban from our community without warning.
+## Consequences
```
IMHO we can drop this section, it's pretty clear from the first paragraph what the consequences will be: "We are committed to providing a welcoming and inspiring community for all and expect our code of conduct to be honored. Anyone who violates this code of conduct may be banned from the community"

searls commented:
:+1: after reading the changes in this PR. I believe having explicit definitions of what behavior is covered and what will (and will not be) responded to can be very useful for resolving disagreements about what constitutes harassment, potentially preventing an incident from escalating further (e.g. pointing to an event's URL and showing in plain language that a similar situation is explicitly mentioned in the CoC).

fat16 committed:
Word tweaks

fat16 committed:
:art:

fat16 committed:
This is said upfront

ammeep commented:
Hey folks - I've made some updates based on the feedback. I'm feeling like this is starting to shape up. 

Let me know what you think :sparkles:

Addvilz commented on index.md:
```diff
@@ -11,18 +11,37 @@ Our open source community strives to:
+
+Our open source community prioritizes marginalized people’s safety over privileged people’s comfort. We will not act on complaints regarding:
+
+- ‘Reverse’ -isms, including ‘reverse racism,’ ‘reverse sexism,’ and ‘cisphobia’
```
I am sorry, but I still have to object against including this. I believe racism is racism, sexism is sexism and phobias are phobias, and it does not matter who are they directed towards.

As an example, if someone would state that I should not participate in certain discussion about whatever topic because I am a male or white, I consider that sexist and racist but with this I would be dismissed simply because I happened to be born with particular skin color and type of genitalia, hence it leads me to conclude including this is in a CoC is biased towards rights of certain groups, and contradicts previous statement in CoC as quoted:

> Although we will fail at times, we seek to treat everyone both as fairly and equally as possible.

ammeep replied:
This is a critical part of this PR and I will not be removing it.

I will do my best to put into words why this is important, and beyond that I do urge you to do some research, free of an innate feeling of being attacked. Beyond that I do not wish debate this topic further.

> As an example, if someone would state that I should not participate in certain discussion about whatever topic because I am a male or white

Everyone is invited to participate in the conversation, but when you come from a position of privilege (which as white people, both you and I do) we have a responsibility to realise the societal benefits we have been afforded. We have a responsibility to realise that minorities are not afforded these same benefits.

As members of a privileged demographic - it is very easy for us to _not_ see inequality. To turn a blind eye. It is very _easy_  to think the playing field is level. 'It doesn't happen to me, so it can't be real'. This is erasure. 

When we claim that the playing field is equal, or level for all, we are erasing the pain and struggles of those who are not afforded the same societal benefits as us. 

When we claim any reverse-sims, what we are really doing is derailing the conversation. Making it about the privileged again. This is erasure.

Life is hard, but some have it harder than others.

Addvilz replied:
Your last sentence is true, as is most of what you said. In fact so true, that it essentially proves my point - but having a hard life does not really justify being racist or sexist. Having said that, I have conducted some research, not limited to academic research or other peoples opinions, but also personal experiences from myself and people around me from wide variety of cultural backgrounds and communities all around the world. What you are essentially saying, instead of not having racism or sexism as such, let's just allow racism and sexist for a minority - because, what could possibly go wrong with privileging one part of society against another. Right? 

This "reverse-ism" thing is based primarily on assumption that in all the communities across the globe it is, lets say, western white man who has the privilege and in general, a white man tends to live happy, full life while everyone else's is just struggle for survival. However, I believe large parts of Asia, Middle east, Africa and Latino-America will probably disagree with you - especially in some culturally established matriarchies. In some places privileged are those who have born in some particular place instead of another, in some places privilege is derived from your profession or many different other factors. Hence, this kind of exclusionism introduces not only ambiguity, but also certain level of discrimination.

Hence again, I believe this very statement excludes a large part of the community essentially saying "you can only complain if you are not a part of stereotypical minority". Do we want a statement like that in a code that should facilitate openness and fair treatment for everyone?

In addition, this is a review. Please, refrain yourself from making it personal in this or any other way, and instead of appealing the reviewer, argue against the argument. Everybody you meet here does not come from utopian  patriarchal society ruled by a stereotypical white male having every white male essentially spending life drinking mojitos and getting a tan.

stephbwills replied:
Just wanted to quickly add that racism and sexism don't define discrimination against one particular race or gender. Privilege vs. non-privilege is really at the heart of the -isms we're discussing, and this is explicitly stated in the preceding paragraph that starts with "our open source community..." I've interpreted this paragraph as a call to administrators who adopt the CoC to protect people who experience institutionalized oppression in their communities.

modality replied:
> This "reverse-ism" thing is based primarily on assumption that in all the communities across the globe it is, lets say, western white man who has the privilege and in general, a white man tends to live happy, full life while everyone else's is just struggle for survival.

Reverse racism and reverse sexism don't exist because racism and sexism are institutional forms of oppression. Context is everything, and the open source community, which is what this Code of Conduct is for (please don't derail), is [overwhelmingly white and male](http://readwrite.com/2013/12/11/open-source-diversity). Case in point: check out the [list of contributors](https://github.com/todogroup/opencodeofconduct/graphs/contributors) to this project.

> However, I believe large parts of Asia, Middle east, Africa and Latino-America will probably disagree with you - especially in some culturally established matriarchies.

This is a cool story. Thank you. Again, context is everything, so discrimination against white men in these communities, if it happened, wouldn't be reverse racism or reverse sexism. It would be racism and sexism, because things like political boundaries and cultural power structures are institutions, and racism and sexism are institutional forms of oppression. However, appealing to examples of matriarchies is derailing this conversation. The open source community is overwhelmingly white and male, so cases of discrimination against women or persons of color (and all the other usual [axes of oppression](http://judge-me-not.weebly.com/uploads/1/0/4/4/10440246/8561435_orig.jpg)) are particularly prevalent.

> In addition, this is a review. Please, refrain yourself from making it personal in this or any other way, 

[Derailing (using emotion)!](http://www.derailingfordummies.com/derail-using-emotion/)

> and instead of appealing the reviewer, argue against the argument. Everybody you meet here does not come from utopian patriarchal society ruled by a stereotypical white male having every white male essentially spending life drinking mojitos and getting a tan.

You're right, everyone doesn't come from the same place, but this isn't a code of conduct for where people come from, it's a code of conduct for _this space_, which is open source software, which is overwhelmingly white and male (and other things too). Context is everything!

Addvilz replied:
Okay, but if that is indeed the case, it should be stated clearly and unambiguously.

I do really like the sentence "[We] call to administrators who adopt the CoC to protect people who experience institutionalized oppression in their communities." - it is clear, with obvious positive intent and not exclusionist. I would very welcome including this statement in some form instead of statement that explicitly suggests no action will be taken in this particular regard on complaints from one group exclusively just because they happen to belong to one.

Addvilz replied:
@modality thank you for your opinion. I would like to refer you to visit communities like Alibaba OS, Taocode, then Rails Girls and similar to gain better insight to the status of open source community in the world, not only English speaking part of it.

> Derailing (using emotion)!

 I am simply stating ad hominem and ergo decedo. My statement involves no emotion whatsoever. As to my wording - I am not a native English speaker, if my statement was formulated in a way that you perceived it as anything different than pointing out the issue with argumentation, that was by no means intended.

Addvilz commented on index.md:
```diff
@@ -11,18 +11,37 @@ Our open source community strives to:
+
+- Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, race, age, regional discrimination, political or religious affiliation
+- Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment
+- Deliberate misgendering or use of ‘dead’ or rejected names
```
As previously commented, this still should be elaborated.

ammeep replied:
@stephbwills you are a word smith - could you please help me elaborate on this?

How about

> Deliberate misgendering. This includes persistently using an incorrect pronoun to describe a person.

I'm worried about use of the word 'incorrect' here. I want it to be clear that the person being addressed get's to decide what is correct or not. Thoughts?

>  or use of ‘dead’ or rejected names. When addressing someone, you must use the name that they feel comfortable being addressed as.

searls replied:
Or maybe "request to be addressed as", since it's impossible to read the minds of someone as to their comfort.

Addvilz replied:
Related to #29 

And I do agree with @searls - people are generally no mind readers, hence, this should include some form of "on request".

stephbwills replied:
@ammeep I feel like the use of "deliberate" covers the cases in which people just get it wrong. I've seen "incorrect" or "wrong" pronoun before, but I made a suggestion below. I also think "on request" puts the onus entirely on the person who was misgendered to make the correction. What about:

> Deliberate misgendering. This includes deadnaming or persistently using a pronoun that does not correctly reflect a person's gender identity. You must address people by the name they give you when not addressing them by their username or handle. 

Also added the bit about handle/username to make it a bit more applicable to the context. Would be good to get some additional community feedback on these definitions, too.

Addvilz replied:
Sounds a lot better and clear. However, the term "‘dead’ or rejected names" is still not explained - the terminology used should be reasonably clear. After spending quite some time searching for definition to this term, I am yet to discover one, hence, this is not clear at all.

stephbwills replied:
More information on deadnaming is available by [google search](https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=deadnaming). 

I sort of feel that if you see something you're unfamiliar with in a CoC, you should check it out, but that's just me. I updated my comment above with "deadnaming," which appears better indexed for search, even if it provides fewer context clues. That's about as far as I'm willing to go, though, without additional input.

Addvilz replied:
Problem with "checking it out" is ambiguity - if there is something to be stated, it must be stated clearly enough to provide reasonable understanding - in best case scenario, without spending hours googling for badly indexed terminology. :) Plus, I might end up finding definition which is not intended in the given reference.

Maybe we could include some sort of glossary with broader explanations for these terms? What do you think?

fat16 committed:
update deadnaming section

ammeep commented:
This is ready for another pass of :eyes: 

@todogroup/owners I would love this to be merged by one of you if there is nothing else outstanding :sparkles:

Addvilz commented:
Before doing that it would be really nice if you would address all the comments and issues pointed out in your pull request you have intentionally ignored so far. Including those between commits.

ammeep commented:
Your feedback has been noted an acted on in many areas of this PR and I am thankful for your input :sparkles:

Unless there are further comments to improve clarity, but not change intent, there will be no further changes to the reverse-ism sections. The intent is deliberate, and I strongly disagree it should be altered to be more lenient to priveged groups.

searls commented:
@addvilz:

this comment (emphasis mine):

> Before doing that it would be really nice if you would address all the comments and issues pointed out in your pull request **you have intentionally ignored so far**. Including those between commits.

Reads as inappropriately presumptive and hostile. I don't think many of the criticisms of this PR have been all that substantive, and I've never seen a healthy project for which it was incumbent upon the person requesting a patch to catalog their reaction to each and every piece of feedback.

Addvilz commented:
@searls I am not requesting reactions, nor am I requesting cataloging them, I am requesting that the issues I am pointing out be addressed based on the objective argument and whatever other backing they are based on, which so far has been mostly, but not exclusively, met by some sort of twisted, almost lobbyist, rhetoric based on subjective opinion and thesis of one particular social activist group completely dismissing any reasoning anyone else might have.  

As to healthy projects, I am not sure I have ever seen any healthy project where, for example, contributor is allowed to introduce bugs to the code just because of particular affiliation or just because the person requesting patch disagrees with it. This, for years, has been the very essence of open source and open contribution. One can not, of course, be expected to please everyone, and it is not needed - however, backing up your argument with something more than an opinion based solely on presumptions without any real backing is a must have if the community is truly open.

As to inappropriately hostile, I for one think ignoring whatever one might have to say only because you disagree with it, is what is hostile and presumptive.

bkeepers commented:
Everyone please chill. I know we all care deeply and may get a little passionate at times.

@Addvilz you've had some great feedback so far. If there are some things you still aren't satisfied with, would you be willing to open another PR (just base it off the `add-back-explicit-definitions` branch, and created it against that branch instead of `master`)?

Addvilz commented:
@bkeepers sure, I can do that.

bkeepers commented:
Thanks @ammeep for your work on this! It feels like this is read to go.

ammeep commented:
![3056872c-eb72-11e3-9ef2-a02704d4e3e5](https://cloud.githubusercontent.com/assets/1024672/8970460/a723225c-35fe-11e5-8f81-19019b8cd990.gif)

stephbwills commented:
@ammeep :sparkles: :sparkles: :sparkles: :sparkles:

NuckChorris commented:
I was absolutely in love with this CoC until I reached the part about reverse-isms and despaired over how rigid and western-centric it was.  Thanks for adding the list of definitions though, that was absolutely necessary, and I'm glad to see that it's all being discussed.

I want to give a huge +1 on the "call to administrators who adopt the CoC to protect people who experience institutionalized oppression in their communities" concept, though I worry that "call" makes it too terribly weak.

I'd also like to bring up (perhaps a bit too late, since this got merged), the vagueness of "We will not act on complaints regarding [...] Refusal to explain or debate social justice concepts" — I originally misinterpreted this as "If you complain that somebody is trying to debate you, we'll do nothing" and that's probably not the intention.  Somebody explained to me that it probably meant "nobody is required to explain a social justice concept to anybody", which is great, but I would like to suggest rewording.  I'm no author, and frankly have no clue how to reword that, but I'm open to suggestions.

Addvilz commented:
@bkeepers I was kind of expecting for this to be on hold till I get the changes I suggested in writing, but oh well...

bkeepers commented:
@Addvilz still looking forward to your changes, but it was important that we got the explicit definitions added sooner rather than later.

MrStonedOne commented:
I'll take what I said in #56 and repeat it here, for the record:

>Regardless of any discussion or debate on rather reverse racism exists, or any discussion or debate on the exact definition of discrimination, language that excludes entire classes from anti-discrimination clauses creates an unwelcoming, impolite, and potentially hostile environment.

just3ws commented:
Who is the arbiter of privilege? Who makes the decision about when racism or sexism or whatever is okay and when it isn't? That puts a lot of pressure on community leaders that isn't fair. This should be about protecting everyone's ability to participate in the community. It should not be about taking people who have arbitrarily been deemed as privileged down a peg or two.
